src/gui/ogr/qgsnewogrconnection.cpp

```diff
@@ -138,4 +139,9 @@ void QgsNewOgrConnection::on_btnConnect_clicked()
   testConnection();
 }
 
+void QgsNewOgrConnection::helpRequest()
+{
+  QgsHelp::openHelp( "managing_data_source/opening_data.html#connecting-to-ogr" );
```

Comment:
This link does not exist. And I don't know to which dialog this button belongs to point you to the right link in doc